### PR TITLE
Stats: add script to build translations in teamcity

### PR DESCRIPTION
**Pull request by kangzj**
#### Proposed Changes

The ability to build language files for Odyssey was added in #71914. The PR proposes to add the building of translations to teamcity.

#### Testing Instructions

* The PR can only be verfied after merged, so IMHO it only needs some proof reading by the platform team

#### Pre-merge Checklist

- [ ] Has the general commit checklist been followed? (PCYsg-hS-p2)
- [ ] [Have you written new tests](https://wpcalypso.wordpress.com/devdocs/docs/testing/index.md) for your changes?
- [ ] Have you tested the feature in Simple (P9HQHe-k8-p2), Atomic (P9HQHe-jW-p2), and self-hosted Jetpack sites (PCYsg-g6b-p2)?
- [ ] Have you checked for TypeScript, React or other console errors?
- [ ] Have you used memoizing on expensive computations? More info in [Memoizing with create-selector](https://github.com/Automattic/wp-calypso/blob/trunk/packages/state-utils/src/create-selector/README.md) and [Using memoizing selectors](https://react-redux.js.org/api/hooks#using-memoizing-selectors) and [Our Approach to Data](https://github.com/Automattic/wp-calypso/blob/trunk/docs/our-approach-to-data.md)
- [ ] Have we added the "[Status] String Freeze" label as soon as any new strings were ready for translation (p4TIVU-5Jq-p2)?
- [ ] For changes affecting Jetpack: Have we added the "[Status] Needs Privacy Updates" label if this pull request changes what data or activity we track or use (p4TIVU-ajp-p2)?

Related to #

### CODE REVIEW

File: .teamcity/_self/projects/WPComPlugins.kt [hunk 1]
```diff
@@ -158,6 +158,17 @@ private object OdysseyStats : WPComPluginBuild(
 	pluginSlug = "odyssey-stats",
 	archiveDir = "./dist/",
 	docsLink = "PejTkB-3N-p2",
+	buildSteps = {
```
Comment (kangzj) [inline .teamcity/_self/projects/WPComPlugins.kt:161]: I assumed the step would be run after `yarn build`. Please do correct me if I'm wrong thanks friends 😄

Comment (kangzj) [reply .teamcity/_self/projects/WPComPlugins.kt:161]: Ah it looks like so:

https://github.com/Automattic/wp-calypso/blob/b44817419dc5b84ec704c86c5180d3c423cbf860/.teamcity/_self/lib/wpcom/WPComPluginBuild.kt#L116

**Review by noahtallen — APPROVED** [review]
Yep, this looks right to me! I also see a /languages folder in the teamcity artifact, so it looks like it's working :)

**Comment by kangzj** [comment]
@noahtallen Thanks for the review 👍 Let's give it a go :shipit:

**Comment by kangzj** [comment]
It works well. Thanks so much @noahtallen 🤝

**Pull request merged**